src/browser/api/system.js

```diff
@@ -492,6 +496,12 @@ exports.System = {
     },
 
     showChromeNotificationCenter: function() {},
+    startCrashReporter: function(identity, options) {
+        const configUrl = coreState.argo['startup-url'];
+
+        crashReporter.startOFCrashReporter(Object.assign({ configUrl }, options));
```

Comment:
Better to assign `Object.assign({ configUrl }, options)` to a variable, for easier future code step-through